lxc/list.go

```diff
@@ -92,18 +92,19 @@ Pre-defined column shorthand chars:
   f - Base Image Fingerprint (short)
   F - Base Image Fingerprint (long)
 
-Custom columns are defined with "key[:name][:maxWidth]":
-  KEY: The (extended) config key to display
+Custom columns are defined with "[config:|devices:]key[:name][:maxWidth]":
+  KEY: The (extended) config or devices key to display. If [config:|devices:] is omitted then it defaults to config key.
   NAME: Name to display in the column header.
```

Comment:
@stgraber I updated help section as well, hope this is fine.

Reply:
Yup, looks good!